### DISCUSSION TITLE
[Question][Hw1] failing of substitution crack?

### DISCUSSION THREAD

**Discussion by ComeToMiCasa** [prosyslab-classroom/cs348-information-security#671] [Q&A]
Name: RIBin Jung

What does failing substitution crack mean? For example, the similarity of the cracked cipher and original text is below k (0 <= k <= 1) etc. My crack is identical to the example in the README but I still fail the crack testcase.

**Comment by ICubE01** [comment]
Answer of #236 indicates that failing vigenere crack test means that at least one of your answer got zero point. I'm not sure, but substitution crack might have a same pass/fail decision.

**Comment by yeonhee-ryou** [comment]
@ICubE- is right. substitution crack have the same pass/fail check.

There are hidden test cases for corner case, and memory/time limits. Your implementation can fail on hiden tests even though it works well with the given examples.
